### DISCUSSION TITLE
Airflow logs copied to GCS but not reflecting in Airflow UI

### DISCUSSION THREAD

**Discussion by tusharraichand** [airflow-helm/charts#836] [Questions & Answers]
I have deployed the latest airflow using helm and I have configured the logs to be pushed to GCS. I can see that logs are being pushed to GCS bucket however Airflow UI is not reflecting them.
This is what I see in UI logs

`*** No logs found in GCS; ti=%s <TaskInstance: xyz>
*** Could not read served logs: [Errno -2] Name or service not known`

My values.yaml for GCS configuration looks like below

    # Airflow scheduler settings
    scheduler:
      extraVolumeMounts:
        - name: google-cloud-key
          mountPath: /opt/airflow/secrets
      extraVolumes:
        - name: google-cloud-key
          secret:
            secretName: sc-key
    
    # Airflow webserver settings
    webserver:
      extraVolumeMounts:
        - name: google-cloud-key
          mountPath: /opt/airflow/secrets
      extraVolumes:
        - name: google-cloud-key
          secret:
            secretName: sc-key

    workers:
      extraVolumeMounts:
        - name: google-cloud-key
          mountPath: /opt/airflow/secrets
      extraVolumes:
        - name: google-cloud-key
          secret:
            secretName: sc-key
    
    # Airflow triggerer settings
    triggerer:
      extraVolumeMounts:
        - name: google-cloud-key
          mountPath: /opt/airflow/secrets
      extraVolumes:
        - name: google-cloud-key
          secret:
            secretName: sc-key
    
    config:
      logging:
        remote_logging: 'True'
        remote_base_log_folder: 'gs://airflow-bucket/logs/'
        remote_log_conn_id: 'sc-key'
        google_key_path:  "/opt/airflow/secrets/key.json"

    connections:
      - id: sc-key
        type: google_cloud_platform
        description: my GCP connection
            extra: |
          {
            "extra__google_cloud_platform__key_path": "/opt/airflow/secrets/key.json",
            "extra__google_cloud_platform__num_retries": 5
          }
    env:
      - name: GOOGLE_APPLICATION_CREDENTIALS
        value: "/opt/airflow/secrets/key.json"
      - name: AIRFLOW__LOGGING__REMOTE_LOGGING
        value: "True"
      - name: AIRFLOW__LOGGING__REMOTE_LOG_CONN_ID
        value: "sc-key"
      - name: AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER
        value: "gs://airflow-bucket/logs/"
      - name: AIRFLOW__CORE__GOOGLE_KEY_PATH
        value: "/opt/airflow/secrets/key.json"

    extraPipPackages:
      - apache-airflow[google]

Can someone please help me with what I am missing here?
Thanks

**Comment by vineethvijay7** [comment]
Thanks for sharing your config. It works for me - but only difference I have is that I use the serviceAccount key json directly in the airflow connection. And I use only the `config:` , no ENV vars.
